support-frontend/assets/helpers/abTests/__tests__/abtestTest.ts

```diff
@@ -21,840 +18,303 @@ jest.mock('ophan', () => ({
-				seed: 0,
-			},
-		};
-		const country = 'US';
-		const countryGroupId = UnitedStates;
 		const participations: Participations = abInit(
 			country,
 			countryGroupId,
-			emptySettings as Settings,
-			tests as Tests,
+			emptySettings,
+			tests,
+			mvt,
 		);
-		const expectedParticipations: Participations = {
-			mockTest: 'notintest',
-		};
-		const expectedMediaQuery = '(min-width:740px)';
+
+		const expectedMediaQuery = '(min-width:740px) and (max-width:980px)';
+
 		expect(window.matchMedia).toHaveBeenCalledWith(expectedMediaQuery);
-		expect(participations).toEqual(expectedParticipations);
+		expect(participations).toEqual({});
 	});
 
-	it('A post-deployment test user should not be allocated into a test', () => {
+	it('does not assign a post-deployment test user to a test', () => {
 		const postDeploymentTestCookie = '_post_deploy_user=true; path=/;';
 
 		function deleteCookie() {
 			document.cookie = `${postDeploymentTestCookie} expires=Thu, 01 Jan 1970 00:00:00 GMT;`;
 		}
 
 		document.cookie = postDeploymentTestCookie;
-		document.cookie = 'GU_mvt_id=12346';
+
 		const tests = {
-			mockTest: {
-				type: 'OTHER',
-				variants: [
-					{
-						id: 'control',
-					},
-					{
-						id: 'variant',
-					},
-				],
-				audiences: {
-					GB: {
-						offset: 0,
-						size: 1,
-					},
-				},
-				isActive: true,
-				referrerControlled: false,
-				seed: 2,
-			},
+			t: buildTest({}),
 		};
-		const country = 'GB';
-		const countryGroupId = GBPCountries;
+
 		const participations: Participations = abInit(
 			country,
 			countryGroupId,
-			emptySettings as Settings,
+			emptySettings,
 			tests,
+			mvt,
 		);
-		const expectedParticipations: Participations = {
-			mockTest: 'notintest',
-		};
-		expect(participations).toEqual(expectedParticipations);
-		deleteCookie();
-	});
 
-	it('Does not allocate a locally rendered epic user into the RemoteEpicVariants AB test', () => {
-		const url = `/test.html?acquisitionData=${encodeURI(
-			JSON.stringify(acquisitionDataMockTestControl),
-		)}`;
-		window.history.pushState({}, 'Test Title', url);
-		const participations: Participations = abInit(
-			'GB',
-			GBPCountries,
-			emptySettings as Settings,
-			{},
-		);
-		expect(participations.RemoteEpicVariants).toBe(undefined);
-	});
+		expect(participations).toEqual({});
 
-	it('Allocates a remotely rendered epic user into the RemoteEpicVariants AB test', () => {
-		const data = {
-			isRemote: true,
-			...acquisitionDataMockTestControl,
-		};
-		const url = `/test.html?acquisitionData=${encodeURI(JSON.stringify(data))}`;
-		window.history.pushState({}, 'Test Title', url);
-		const participations: Participations = abInit(
-			'GB',
-			GBPCountries,
-			emptySettings as Settings,
-			{},
-		);
-		expect(participations.RemoteEpicVariants).toBe('remote');
+		deleteCookie();
 	});
-});
 
-describe('Correct allocation in a multi test environment', () => {
-	afterEach(() => {
-		window.localStorage.clear();
-	});
```

Comment:
I deleted all of these tests. They're quite complex and hard to figure out what they're actually testing. I belive, when you drill down what they're actually doing is testing that the audience targeting works with the `offset` and `size`. I've replaced them all with two simpler tests that explicitly test that behaviour